File: OOP.py
class Person:
    def __init__(self,name,age,gender):
        self.name=name
        self.age=age
        self.gender=gender

    def introduce(self):
        print("Your name is :",self.name,",",end=" ")
        print("You are  :",self.age ,"years old ,",end=" ")
        print("You identify as a :",self.gender,".")

man1=Person('Njoki',23,'Female')
man1.introduce()

File: OOP.dart
class Student{
  String name;
  int age;
  String grade;

  Student(this.name,this.age,this.grade);

  void displayInfo(){
    print("Your name is $name");
    print("Your age is $age");
    print("Your are in  grade $grade");

  }
}
class Teacher{
  String name;
  int age;
  String subject;

  Teacher(this.name,this.age,this.subject);

  void displayInfo(){
    print("Your name is $name");
    print("Your age is $age");
    print("You teach $subject");

  }
}
class Combined{
  void createNprintInstances(){
  var student=Student('Martha',12,'Four');
  var teacher=Teacher('Joseph',34,'Kiswahili');

  print("Student Information");
  student.displayInfo();
   print("Teacher Information");
  teacher.displayInfo();

  }
  

}
void main(){
 var combined=Combined();
 combined.createNprintInstances();
}